addons/calendar/static/tests/attendee_calendar_views_tests.js

```diff
@@ -0,0 +1,258 @@
+            res_model_name: 'Time Off',
+
+        });
+
+        await makeView({
+            type: "calendar",
+            resModel: "event",
+            serverData,
+            arch: `
+                <calendar js_class="attendee_calendar" 
+                          event_open_popup="1" 
+                          date_start="start" 
+                          date_stop="stop" 
+                          all_day="allday" 
+                          mode="month">
+                              <field name="partner_ids" options="{'block': True, 'icon': 'fa fa-users'}"
+                                     filters="1" widget="many2manyattendeeexpandable" write_model="filter_partner"
+                                     write_field="partner_id" filter_field="partner_checked" avatar_field="avatar_128"
+                              />
+                              <field name="partner_id" string="Organizer" options="{'icon': 'fa fa-user-o'}"/>
+                              <field name="user_id"/>
+                              <field name="start"/>
+                              <field name="stop"/>
+                              <field name="allday"/>
+                              <field name="res_model_name" invisible="not res_model_name" 
+                                     options="{'icon': 'fa fa-link', 'shouldOpenRecord': true}"
+                              />
+                          </calendar>
+            `,
+            RPmockC(route, args) {
+                if (route === "/web/dataset/call_kw/res.users/has_group") {
+                    return Promise.resolve(true);
+                } else if (route === "/web/dataset/call_kw/res.partner/get_attendee_detail") {
+                    return Promise.resolve([]);
+                } else if (route === '/web/dataset/call_kw/calendar.event/action_open_calendar_event') {
+                    return Promise.resolve({
+                        "type": "ir.actions.act_window",
+                        "res_model": "hr.leave",
+                        "views": [[false, "form"]],
+                        "target": "current",
+                        "res_id": 1,
+                        "context": {
+                            "lang": "en_US",
+                            "tz": "Europe/Bruxelles",
+                            "uid": 2,
+                            "allowed_company_ids": [1]
+                        }
+                    });
+                }
+            },
+        });
+        assert.containsOnce(
+            target,
+            ".o_calendar_renderer .fc-view-container",
+            "should instance of fullcalendar"
+        );
+        await changeScale(target, 'week');
+        await clickEvent(target, 3);
+        assert.containsOnce(
+            target,
+            ".fa-link",
+            "A link icon should be present"
+        );
+        assert.strictEqual(target.querySelector("li a[href='#']").innerHTML, "Time Off");
```

Comment:
prefer textContent rather than innerHTML
```suggestion
        assert.strictEqual(target.querySelector("li a[href='#']").textContent, "Time Off");
```